bin/ingress_nginx.py

```diff
@@ -87,6 +89,8 @@ def get_patch_directory(version: str, ingress_rootdir: str) -> str:
         "v1.11.2": f"{ingress_rootdir}/images/nginx-1.25/rootfs/patches",
         "v1.11.1": f"{ingress_rootdir}/images/nginx-1.25/rootfs/patches",
         "v1.11.0": f"{ingress_rootdir}/images/nginx-1.25/rootfs/patches",
+        "v1.10.6": f"{ingress_rootdir}/images/nginx/rootfs/patches",
+        "v1.10.5": f"{ingress_rootdir}/images/nginx/rootfs/patches",
```

Comment:
```suggestion
        "v1.10.6": f"{ingress_rootdir}/images/nginx-1.25/rootfs/patches",
        "v1.10.5": f"{ingress_rootdir}/images/nginx-1.25/rootfs/patches",
```

Would these changes make sense ?

Reply:
What's your thought process?

Reply:
This is intentional. I recommend taking a look at the code to better understand the underlying logic. If anything remains unclear, I’m happy to discuss it further.